.github/workflows/node.js.yml

```diff
@@ -0,0 +1,33 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Build, Test, Lint
+
+on:
+  push:
+    branches: [ main, exercise2 ]
+  pull_request:
+    branches: [ main, exercise2 ]
+
+defaults:
+  run:
+    working-directory: ./webapp/dojo_news
+
+jobs:
+  build-test-lint:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x]
```

Comment:
Any reason why you choose three versions instead of specifying one?

Reply:
These are the [three current LTS versions](https://nodejs.org/en/about/releases/).

Reply:
On the other PRs I said one version would be enough as you have control over the node version: Either you on your laptop or your build server will transpile your VueJS code.